ReadMe.md

```diff
@@ -25,10 +25,13 @@ variable that depends on state or user input, using Safe Timers is a good idea.
 
 ## API
 
-**Timer setTimeout(Function callback, number interval, mixed arg1, mixed arg2, ...)**
+### Timeout
 
-Calls `callback` after at least `interval` milliseconds have passed. All arguments passed after the `interval` will be
-passed to the callback once it gets invoked. Returns a `Timer` instance.
+There are two ways to create a new `Timeout` instance: `setTimeout` and `setTimeoutAt`.
+
+**Timeout setTimeout(Function callback, number interval, mixed arg1, mixed arg2, ...)**
+
+- Creates and returns a `Timeout` instance that will call `callback` after at least `interval` milliseconds have passed. All arguments passed after the `interval` will be passed to the callback once it gets invoked.
```

Comment:
I would recommend losing the `- ` characters at the start of this line as they are causing bulleted list formatting for the description of the method. The formatting looks odd and the indentation doesn't really line up well with the code examples and sometimes present subsequent non-bulleted supplemental sentences (e.g. about how to clear `Timeout`/`Interval` objects).

Reply:
Noted. I was trying to establish a distinction between the text explaining the overall API usage versus the text detailing the specifics of each function, but if you prefer it this way I'll leave it unindented.